controllers/contacts-controllers.js

```diff
@@ -0,0 +1,65 @@
+import contactsService from "../models/contacts.js";
+import { HttpError } from "../helpers/index.js";
+import Joi from "joi";
+import ctrlWrapper from "../decorators/contacts-decorator.js";
+
+const contactAddSchema = Joi.object({
+  name: Joi.string().required(),
```

Comment:
schema should be in separate file. You should validate email and phone properly

Reply:
Thanks for your comments, I've already fixed according to your notification